### PR TITLE
refactor: add `_y` suffix to names of yearly variables

### PR DESCRIPTION
**Pull request by lars-reimann**
### What problem do you want to solve?

Closes partially #575.

* Add `_y` suffix to names of yearly variables (breaking change)

### Todo

- [X] Pick an appropriate title.
- [X] Put `Closes #XXXX` in the first PR comment to auto-close the relevant issue once
      the PR is accepted. This is not applicable if there is no corresponding issue.
- [X] Document PR in CHANGES.md.

### CODE REVIEW

**Comment by lars-reimann** [comment]
Since this PR is already quite lengthy but self-contained, I'll do the 2nd and 3rd step of #575 in a separate PR.

File: docs/geps/gep-04.md [hunk 1]
```diff
@@ -80,44 +80,44 @@ is able to replace this function with her own version.
...
     """
-    return abgelt_st_params["satz"] * zu_verst_kapitaleink_tu
+    return abgelt_st_params["satz"] * zu_verst_kapitaleink_y_tu
 ```
 
-The function {func}`abgelt_st_tu` requires the variable `zu_verst_kapital_eink_tu` which
-is the amount of taxable capital income on tax unit level (the latter is implied by the
-`_tu` suffix, see {ref}`gep-1`). `zu_verst_kapital_eink_tu` must be provided by the user
-as a column of the input data or it has to be the name of another function.
+The function {func}`abgelt_st_y_tu` requires the variable `zu_verst_kapital_eink_tu`
```
Comment (hmgaudecker) [inline docs/geps/gep-04.md:100]:
```suggestion
The function {func}`abgelt_st_y_tu` requires the variable `zu_verst_kapital_eink_y_tu`,
```

File: docs/geps/gep-04.md [hunk 2]
```diff
@@ -80,44 +80,44 @@ is able to replace this function with her own version.
...
+which is the amount of taxable capital income on tax unit level (the latter is implied
+by the `_tu` suffix, see {ref}`gep-1`). `zu_verst_kapital_eink_tu` must be provided by
```
Comment (hmgaudecker) [inline docs/geps/gep-04.md:102]:
```suggestion
by the `_tu` suffix, see {ref}`gep-1`). `zu_verst_kapital_eink_y_tu` must be provided by
```

File: src/_gettsim/parameters/grunds_im_alter.yaml [hunk 3]
```diff
@@ -61,11 +61,11 @@ ges_rente_anr_frei:
       rate: 0.3
 kapitaleink_anr_frei:
   name:
-    de: Freibetrag des Kapitaleinkommens für Grundsicherung
+    de: Freibetrag des kapitaleink_yommens für Grundsicherung
```
Comment (hmgaudecker) [inline src/_gettsim/parameters/grunds_im_alter.yaml:64]:
```suggestion
    de: Freibetrag des Kapitaleinkommens für Grundsicherung
```

File: src/_gettsim/parameters/grunds_im_alter.yaml [hunk 4]
```diff
@@ -61,11 +61,11 @@ ges_rente_anr_frei:
...
     en: Amount of capital income not subject to transfer withdrawal
   description:
     de: >-
-      Nur das Kapitaleinkommen, das über diesem Freibetrag liegt, wird auf die
+      Nur das kapitaleink_yommen, das über diesem Freibetrag liegt, wird auf die
```
Comment (hmgaudecker) [inline src/_gettsim/parameters/grunds_im_alter.yaml:68]:
```suggestion
      Nur das Kapitaleinkommens, das über diesem Freibetrag liegt, wird auf die
```

**Review by hmgaudecker — APPROVED** [review]
Perfect! Just a couple of omissions in the docs and cases where the replacing was too greedy.

One thing that would be great would be to adjust the text of GEP 1 where we talk about the default being yearly.

Thanks a lot!!!

**Pull request merged**